### ISSUE TITLE
link command for ImageMagick 7.1.0-37 with GCCcore/11.3.0 fails

### ISSUE THREAD

**Issue by boegel** [easybuilders/easybuild-easyconfigs#16236]
The `ImageMagick` easyconfig file that was merged as a part of #15670 fails for me currently, with:

```
/software/binutils/2.38-GCCcore-11.3.0/bin/ld: cannot find -lpangocairo-1.0: No such file or directory
/software/binutils/2.38-GCCcore-11.3.0/bin/ld: cannot find -lpango-1.0: No such file or directory
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:7133: MagickCore/libMagickCore-7.Q16HDRI.la] Error 1
```

I'm puzzled what's wrong here, since:
* This easyconfig did work for various others, see test reports in #15670, and it also did work for me a while ago
* The missing Pango libraries being mentioned as there in the `lib` subdirectory of the `Pango/1.50.7-GCCcore-11.3.0` installation directory, and `Pango` is being loaded as a dependency (although indirectly)

The problem occurred for me during the regression test for EasyBuild v4.6.1, when starting to build the whole software tree from scratch...

**Comment by Micket** [comment]
We can probably just disable with --without-pango. Pango dep wasn't added intentionally here.

**Comment by boegel** [comment]
The indirect Pango dependency was introduced by the changes in #16112.

The problem is fixed by adding `Pango` as a direct dependency (which results in adding `$EBROOTPANGO/lib` to `$LDFLAGS`, rather than only having it included in `$LIBRARY_PATH), cfr #16237
